web/pages/api/v0/increment.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  CORS_ORIGIN_MANIFOLD,
+  CORS_ORIGIN_LOCALHOST,
+} from 'common/envs/constants'
+import { NextApiRequest, NextApiResponse } from 'next'
+import { applyCorsHeaders } from 'web/lib/api/cors'
+import * as admin from 'firebase-admin'
+import { auth } from 'web/lib/firebase/users'
+
+export const config = { api: { bodyParser: true } }
+
+function initAdmin() {
+  // TODO: Support dev vs prod environments; maybe private instances too?
+  // TODO: How do we get the Firebase Admin secret to exist here, _without_ leaking it
+  // to PR preview branches from our users? (Maybe Vercel already does this safely?)
```

Comment:
Right, but the operative question is whether e.g. when an open source contributor creates a new PR, does that PR preview have access to the admin secret? My tentative guess is that it will, if we approve the PR 

It could be bad if so -- imagine a malicious contributor changes the preview code to use the admin secret and wipe our database, or steal mana. (or does Vercel force a re-authorization on each new github push? if so, we're fine)

(Also maybe this is an attack vector we're vulnerable to already, hm...)

Reply:
Yeah looks like Vercel does the right thing already in forcing re-authorization on each push: https://github.com/manifoldmarkets/manifold/pull/1106. Another hooray for Vercel!